src/power_grid_model/core/power_grid_model.py

```diff
@@ -242,7 +244,7 @@ def _calculate_impl(  # pylint: disable=too-many-positional-arguments
         options: Options,
         continue_on_batch_error: bool,
         decode_error: bool,
-        experimental_features: _ExperimentalFeatures | str,  # pylint: disable=too-many-arguments
+        experimental_features: _ExperimentalFeatures | str,  # pylint: disable=too-many-arguments,unused-argument
```

Comment:
space afte comma?

Reply:
nope, the pattern that pylint uses is without spaces

Reply:
either way, i'll just remove this cause sonar cloud is also complaining

Reply:
actually, i'll keep it, cause otherwise the experimental check is somewhere else